data/json/items/armor/bandolier.json

```
@@ -262,5 +262,25 @@
     },
     "valid_mods": [ "resized_large" ],
     "flags": [ "WATER_FRIENDLY", "BELTED", "COMPACT" ]
+  },
+  {
+    "id": "bandolier_grenade",
+    "type": "ARMOR",
+    "name": { "str": "grenade bandolier" },
```

For consistency with grenade pouches specifying whether they store hand grenades or grenade ammo
```suggestion
    "name": { "str": "grenade ammo bandolier" },
```